modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc

```diff
@@ -0,0 +1,110 @@
+
+== Pre-requisites
+
+The SR-IOV operator must be installed  and configured in order to obtain virtual functions in a container.
+
+To configure SR-IOV interfaces, an SR-IOV network and policy must be created for each interface.
+
+The SR-IOV operator creates a network attachment definition for each SR-IOV interface, based on the SR-IOV network and policy defined.
+
+== Creating a bond network attachment definition
+
+Now that the SR-IOV virtual functions are available, you can create a bond network attachment definition.
+
+[source,yaml]
+----
+    apiVersion: "k8s.cni.cncf.io/v1"
+    kind: NetworkAttachmentDefinition
+    metadata:
+      name: bond-net1
+      namespace: demo
+    spec:
+      config: '{
+      "type": "bond", <1>
+      "cniVersion": "0.3.1",
+      "name": "bond-net1",
+      "ifname: "bond0" <2>
+      "mode": "active-backup",
+      "failOverMac": 1,
+      "linksInContainer": true,
+      "miimon": "100",
+      "links": [ <3>
+            {"name": "net1"},
+            {"name": "net2"}
+        ],
+      "ipam": {
+            "type": "host-local"
+            "subnet": "10.56.217.0/24"
+            "routes": [{
+            "dst": "0.0.0.0/0"
+            }],
+            "gateway": "10.56.217.1"
+        }
+      }'
+
+----
+
+<1> The type is "bond".
+
+<2> The ifname attribute specifies the name of the bond interface.
+
+<3> The links section defines which interfaces will be used to create the bond. Multus names the attached interfaces as: "net", plus a consecutive number, starting with one.
+
+== Creating a pod using a bond interface
+
+You can now test the setup by creating a pod using a bond interface.
+
+[source,yaml]
+----
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: bondpod1
+      namespace: mm
+      annotations: <1>
+        k8s.v1.cni.cncf.io/networks: mm/sriovnet1, mm/sriovnet2, mm/bond-net <2>
+    spec:
+      containers:
+      - name: podexample
+        image: quay.io/openshift/origin-network-interface-bond-cni:4.9.0
+        command: ["/bin/bash", "-c", "sleep INF"]
+
+----
+
+<1> Note the network annotation.
+
+<2> The network annotation contains two SR-IOV network attachments, and one bond network attachment. The bond attachment uses the two SR-IOV interfaces as bond slaves.
+
+You can inspect the pod interfaces with the following command:
+
+[source,yaml]
+----
+$ oc rsh -n mm bondpod1
+sh-4.4#
+sh-4.4# ip l
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00 <1>
+3: eth0@if799: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1400 qdisc noqueue state UP mode DEFAULT group default
+    link/ether 0a:58:0a:85:02:d8 brd ff:ff:ff:ff:ff:ff link-netnsid 0 <2>
+4: bond0: <BROADCAST,MULTICAST,MASTER,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default qlen 1000 <3>
+
+
+----
+
+<1> The net one interface based on an SR-IOV virtual function.
```

Comment:
Isn't `net1` a field or option? If yes, please use backticks and use the name used in the code/command.